pkg/controller/container-runtime-config/helpers.go

```diff
@@ -448,6 +453,20 @@ func createCRIODropinFiles(cfg *mcfgv1.ContainerRuntimeConfig) []generatedConfig
 	return generatedConfigFileList
 }
 
+func createDefaultContainerRuntimeFile() []generatedConfigFile {
+	var (
+		generatedConfigFileList []generatedConfigFile
+		err                     error
+	)
+	tomlConf := tomlConfigCRIODefaultRuntime{}
+	tomlConf.Crio.Runtime.DefaultRuntime = "runc"
+	generatedConfigFileList, err = addTOMLgeneratedConfigFile(generatedConfigFileList, CRIODropInFilePathDefaultContainerRuntimeRunc, tomlConf)
+	if err != nil {
+		klog.V(2).Infoln(err, "error setting default-container-runtime to crio.conf.d: %v", err)
```

Comment:
error isn't handled here... the function should probably return an error.

Reply:
I don't think this concern has been addressed?